### DISCUSSION TITLE
Disable rebuild RequiredAddons in Mikero tools

### DISCUSSION THREAD

**Discussion by serraedoardo** [acemod/ACE3#8726] [Q&A] [answered]
Hi,

I was trying to set up the ace's development environment and got stuck at the requirements. In particular, I can't figure out how to disable rebuild requiredAddons in MakePbo tool.

Any suggestions? Thank you.

**Comment by jonpas** [comment]
I don't think newer versions have the setting anymore.

@Dahlgren ?

**Comment by Dahlgren** [comment, marked as answer]
If it can be used with your version of mikero tools the rebuild addons is configured via `+F` to enable or `-F` to disable
